generate_test_certs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Does this require bash or would /bin/sh work too?

Reply:
Good point! I updated it to use `/bin/sh` instead.